packages/blitz/src/installer/utils/paths.ts

```diff
@@ -18,12 +18,23 @@ function getBlitzPath(type: string) {
   }
 }
 
+function findPageDir() {
+  const srcPagePath = `src/pages`
+  const srcPage = fs.existsSync(path.resolve(srcPagePath))
+
+  if (srcPage) {
+    return srcPagePath
+  } else {
+    return `pages`
```

Comment:
I think previous versions of blitz were using `app/pages` instead of just `pages`.